src/app/containers/StoryPromo/index.jsx

```diff
@@ -12,15 +13,24 @@ import Timestamp from '../Timestamp';
 
 const StoryPromo = ({ item }) => {
   const headline = deepGet(['headlines', 'headline'], item);
+  const url = deepGet(['locators', 'assetUri'], item);
   const summary = deepGet(['summary'], item);
   const timestamp = deepGet(['timestamp'], item);
   const imageValues = deepGet(['indexImage'], item);
 
   const Image = imageValues && <StoryPromoFigure {...imageValues} />;
 
+  if (!headline || !url) {
+    return null;
+  }
+
   const Info = (
     <Fragment>
-      {headline && <Headline script={latin}>{headline}</Headline>}
+      {headline && (
+        <Headline script={latin}>
```

Comment:
Instead of passing `latin` script directly, could we get it from the service context.

Reply:
Oh whoops, good spot, totally missed that 🙈